urdf2webots/parserURDF.py

```diff
@@ -923,6 +923,8 @@ def isRootLink(link, childList):
 
 def parseGazeboElement(element, parentLink, linkList):
     """Parse a Gazebo element."""
+    if element.hasAttribute("reference"):
```

Comment:
It seems this test was aldready done for the camera and lidar, but not for the IMU (with an extra condition):

```suggestion
    if element.hasAttribute("reference") and element.getAttribute('reference') in linkList:
```

We can them remove this test from lines 942-943 and 967-968.

Reply:
The condition `element.getAttribute('reference') in linkList` does not work since the `linkList` contains `urdf2webots.parserURDF.Link` objects and the attribute is just a string. 
1. The behavior when the sensor is added to the sensor list but the parent link does not exist, is that the sensor is simply not added since https://github.com/cyberbotics/urdf2webots/blob/fe1a61eae935ffe5671e28fdb6be93209b4bbf68/urdf2webots/writeProto.py#L114 is never true.
2. The other option is to check if the link name exists in the list of link elements. If the link does not exist, the sensor would be placed in the base link.

So the code in 942-943 and 967-968 unfortunately never worked as intended.

Would you like to see option 1 or 2?

Reply:
Thank you for the investigation and explanation. Option 2 looks better to me as it creates a useful list earlier.

Reply:
The commit i made has this option implemented.